lib/runner/extensions/event.command.js

```diff
@@ -194,6 +177,27 @@ module.exports = {
                         !assertion.passed && assertionFailed.push(assertion.name);
                     });
 
+                var run = this;
+
+                this.host.on('execution.request', function (cursor, id, requestId, request) {
+                    // clone cursor and add source
+                    var cursorWithSource = _.clone(cursor);
+
+                    cursorWithSource.source = 'script'; // @todo - get script type from the sandbox
+
+                    run.immediate('request', {
+                        item: new sdk.Item({
```

Comment:
@shamasis when creating a new `Item` without a parent reference to any collection, the collection variable resolution flow [here](https://github.com/postmanlabs/postman-runtime/blob/develop/lib/runner/request-helpers-presend.js#L12) breaks. 

Unsure how to handle this.